src/main/java/com/seailz/discordjar/model/channel/GuildChannel.java

```diff
@@ -111,4 +118,65 @@ default void addPermissionOverwrite(@NotNull PermissionOverwrite overwrite) {
     default CreateChannelInviteAction createInvite() {
         return new CreateChannelInviteAction(discordJv(), id());
     }
+
+    /**
+     * Creates an Incoming Webhook for this channel.
+     *
+     * @param name The name of the Webhook.
+     * @return The created Webhook.
+     * @implNote Avatars are not implemented yet.
+     */
+    default IncomingWebhook createIncomingWebhook(String name) throws DiscordRequest.UnhandledDiscordAPIErrorException {
```

Comment:
```suggestion
    default IncomingWebhook createWebhook(String name) throws DiscordRequest.UnhandledDiscordAPIErrorException {
```

Reply:
This is intentional as I plan to add the other types of webhooks as well, this PR just isn't done. Possibly just the channel follower webhook but not sure

Reply:
Channel follower will be done through a separate method and should not be part of this PR, and  application webhooks are only for interactions.

Reply:
Got it, will fix now